Update WriteResult.java

DijonnaWatson opened the pull request:
Making this status checker match the factorComputeResult interface. Are you guys ok with this change?

DijonnaWatson committed:
Update WriteResult.java

making this status checker match the factorComputeResult interface

jeremy23newman commented on src/factorFinder/WriteResult.java:
```diff
@@ -1,10 +1,23 @@
+
+    private final boolean success;
+
+    private WriteResultStatus(boolean success) {
```
nice way easier to understand

DijonnaWatson committed:
Update FactorComputeEngine.java

I just realized I don't think there's a reason for the delimiter to be a method here since it's already defined in the FactorComputeRequest.

DijonnaWatson committed:
Add files via upload

DijonnaWatson committed:
Delete src/factorFinder/FactorComputeEngineImpl.class

wrong file

DijonnaWatson committed:
Update FactorComputeEngine.java

DijonnaWatson committed:
Add files via upload

this was the file i meant to upload earlier

DijonnaWatson committed:
Update FactorComputeEngineImpl.java

what i got so far for the compute engine implementation. I'm also just fixing the indentation

DijonnaWatson committed:
Update FactorComputeEngineImpl.java

ViceRaven committed:
added delimiter logic to FactorComputeEngineImpl.java

ViceRaven committed:
Create FactorComputationCoordinationImpl.java

ViceRaven committed:
added computation logic to FactorComputationCoordinationImpl.java

ViceRaven committed:
forgot to add list import InputConfig.java

DijonnaWatson committed:
Update WriteResult.java

nvm i changed it back to the original since it messes with certain things already using it in other classes

ViceRaven committed:
Revamped FactorComputationCoordinationImpl.java

Original file was way to confusing and didn't have the steps for 2 explicitly outline. Hopefully this revamped version will give more peace of mind and help us follow along coherently.

DijonnaWatson committed:
Update FactorComputeEngineImpl.java

added the default delimiter back

DijonnaWatson committed:
Update FactorComputeEngineImpl.java

ViceRaven committed:
Create FactorDataStoreImpl.java

DijonnaWatson committed:
Update and rename FactorComputationCoordinationImpl.java to FactorComputationCoordinatorImpl.java

fixing package name, and outter file name

DijonnaWatson committed:
Update WriteResult.java

fixed the WriteResult again to match the FactorComputeResult

ViceRaven committed:
Update FactorDataStoreImpl.java with code from eclipse

Retrieves input data from input config
Also contains if-else for when the data is null or not

ViceRaven committed:
Update FactorComputationCordinatorPrototype.java

small change to print out a better confirmation message upon getstatus

ViceRaven committed:
Updated package on WriteResult.java

ViceRaven committed:
added missing import statements FactorComputeRequest.java

ViceRaven committed:
added missing package Update OutputConfig.java

ViceRaven committed:
Update FactorDataStorePrototype.java

had to tweak prototype to work with the coordination implementation

added some things like output configuration, processing loaded data, and checking the write result status.
